### PR TITLE
fix:count in repeatEach so that testcases are distinctive as repeatEach is not the same as retry

### PR DESCRIPTION
**Pull request by alyaothman14**
### Context
Issue link: https://github.com/allure-framework/allure-js/issues/663
image

#### Checklist
- [x] [Sign Allure CLA][cla]
- [x] Provide unit tests

[cla]: https://cla-assistant.io/accept/allure-framework/allure2

### CODE REVIEW

**Comment by alyaothman14** [comment]
@baev I would really appreciate if this can be reviewed and released as the results in AllureTestOps is skewed because of this e.g the pipeline will fail but in testops it marks it as passed

File: packages/allure-playwright/src/index.ts [hunk 1]
```diff
@@ -112,7 +113,7 @@ class AllureReporter implements Reporter {
       .join("/");
 
     const nameSuites = suiteTitles.length > 0 ? `${suiteTitles.join(" ")} ` : "";
-    const fullName = `${relativeFile}#${nameSuites}${test.title}`;
+    const fullName = `${relativeFile}#${nameSuites}${test.title}${repeatEach ? ` (repeat #${test.repeatEachIndex + 1})` : ""}`;
```
Comment (baev) [inline packages/allure-playwright/src/index.ts:116]: Add it as a test parameter instead

Comment (baev) [reply packages/allure-playwright/src/index.ts:116]:
```js
if (project?.repeatEach) {
  allureTest.addParameter("Repeation", test.repeatEachIndex + 1);
}
```

**Review by baev — CHANGES_REQUESTED** [review]
could you please add a test for the change

File: packages/allure-playwright/src/index.ts (hunk 1 above)
Comment (alyaothman14) [reply packages/allure-playwright/src/index.ts]: Thanks updated

**Comment by baev** [comment]
change included to https://github.com/allure-framework/allure-js/pull/676 

Thanks for the contribution!

**Pull request closed without merging**